assets/scss/bootstrap-migration/custom/_navbar.scss

```diff
@@ -1,4 +1,13 @@
 .navbar-search-bar {
-    min-height: 3.25rem;
-    font-size: $font-size-base;
+  min-height: 3.25rem;
+}
+
+.navbar.navbar-dark {
+  li.active > {
+    a,
+    a:hover {
+      color: $body-color;
```

Comment:
Probably will be replaced with correct navbar template

Reply:
Navbar template fixed, but this code isn't replaced, because it's needed for active state